packages/spanner/system-test/spanner.js

```diff
@@ -2835,19 +2833,21 @@ var spanner = new Spanner(env);
           transaction.run(query, function(err, rows) {
             assert.ifError(err);
 
-            var row = rows[0].toJSON();
+            var originalRows = extend(true, {}, rows);
 
+            // Make arbitrary update.
             table.update({
-              Key: row.Key,
-              StringValue: 'v33'
+              Key: rows[0].toJSON().Key,
+              StringValue: 'overridden value'
             }, function(err) {
               assert.ifError(err);
 
               transaction.run(query, function(err, rows_) {
                 assert.ifError(err);
 
-                var row = rows_.pop().toJSON();
-                assert.strictEqual(row.StringValue, 'v3');
+                rows_ = extend(true, {}, rows_);
+
+                assert.deepStrictEqual(rows_, originalRows);
```

Comment:
I believe this assertion is basically "make sure I'm still reading data in an old state". I took out the specific value expectation, "v3", hopefully making it more clear that all rows should be the same as they were before the update.